joomla/joomla-cms
# [5.1] Give a name for the breadcrumbs schema.org asset

### Summary of Changes

The PR gives a name for the breadcrumbs asset for schema.org, 
By default the name is md5 of the asset content, wich is not very useful for this asset.
Specific name will allow others extensions easily access to it.


### Testing Instructions
Apply patch, visit the page with the breadcrumbs.
Check page source, and make sure BreadcrumbList schema is there.


### Actual result BEFORE applying this Pull Request
Works


### Expected result AFTER applying this Pull Request
Works


### Link to documentations
Please select:
